input/input.go

```diff
@@ -73,13 +69,9 @@ func NewDefaultHandler(metrics mdata.Metrics, metricIndex idx.MetricIndex, input
 		receivedMPNO: stats.NewCounter32(fmt.Sprintf("input.%s.metricpoint_no_org.received", input)),
 		// metric input.%s.metricdata.discarded.invalid is a count of times a metricdata was invalid by input plugin
 		invalidMD: stats.NewCounterRate32(fmt.Sprintf("input.%s.metricdata.discarded.invalid", input)),
-		// metric input.%s.metricdata.discarded.invalid_tags is a count of times a metricdata was considered invalid due to
-		// invalid tags in the metric definition. all rejected metrics counted here are also counted in the above "invalid" counter
-		invalidTagMD: stats.NewCounterRate32(fmt.Sprintf("input.%s.metricdata.discarded.invalid_tag", input)),
-		// metric input.%s.metricdata.discarded.invalid_utf is a count of times a metricdata was considered invalid due to
-		// invalid utf8 in either the name or tags in the metric definition. all rejected metrics counted here are also counted in
-		// above "invalid" counter
-		invalidUtfMD: stats.NewCounterRate32(fmt.Sprintf("input.%s.metricdata.discarded.invalid_utf", input)),
+		// metric input.%s.metricdata.discarded.invalid_input is a count of times a metricdata was considered invalid due to
+		// invalid input data in the metric definition. all rejected metrics counted here are also counted in the above "invalid" counter
+		invalidInputMD: stats.NewCounterRate32(fmt.Sprintf("input.%s.metricdata.discarded.invalid_tag", input)),
```

Comment:
```suggestion
		invalidInputMD: stats.NewCounterRate32(fmt.Sprintf("input.%s.metricdata.discarded.invalid_input", input)),
```

Reply:
I think I may have fixed this while you were reviewing it.